### PR TITLE
[PL] add translation of enum lesson

### PR DESCRIPTION
**Pull request by Koziolek**
in scope of issue #202

### CODE REVIEW

**Comment by doomspork** [comment]
@kdebowski could you take a look at this when you have a moment?

File: pl/basic/enum.md [hunk 1]
```diff
@@ -123,7 +122,7 @@ iex> Enum.reduce([1, 2, 3], fn(x, acc) -> x + acc end)
 
 ### sort
 
-Sorting our collections is made easy with not one, but two, `sort` functions.  The first option available to us uses Elixir's term ordering to determine the sorted order:
+Sortowanie kolekcji jest bardzo proste dzięki nie jednej a dwóm funkcjom `sort`.  Pierwsza z nich używa porządkuje elementy zgodnie ze specyfikacją Elixira:
```
Comment (kdebowski) [inline pl/basic/enum.md]: Unwanted word: używa

File: pl/basic/enum.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+```elixir
+iex> Enum.reduce([1, 2, 3], 10, fn(x, acc) -> x + acc end)
+16
+iex> Enum.reduce([1, 2, 3], fn(x, acc) -> x + acc end)
+6
+```
+
+### sort
+
+Sortowanie kolekcji jest bardzo proste dzięki nie jednej a dwóm funkcjom `sort`.  Pierwsza z nich używa porządkuje elementy zgodnie ze specyfikacją Elixira:
```
Comment (kdebowski) [inline pl/basic/enum.md]: Unwanted word: używa

**Comment by kdebowski** [comment]
@Koziolek I've found one unwanted word. Despite that word it's OK!

**Comment by Koziolek** [comment]
Fixed

**Comment by kdebowski** [comment]
@doomspork it's ready to be merged

**Comment by doomspork** [comment]
Thank you @Koziolek & @kdebowski

**Pull request merged**